recipes/pangomm/all/conanfile.py

```diff
@@ -0,0 +1,207 @@
+
+        tools.replace_in_file(
+            os.path.join(self._source_subfolder, "tools",
+                         "extra_defs_gen", "meson.build"),
+            "required: glibmm_dep.type_name() != 'internal',",
+            f"required: glibmm_dep.type_name() != 'internal', dirs: {glibmm_generate_extra_defs_dir}")
+
+        if is_msvc(self):
+            # when using cpp_std=c++NM the /permissive- flag is added which
+            # attempts enforcing standard conformant c++ code
+            # the problem is that older versions of Windows SDK is not standard
+            # conformant! see:
+            # https://developercommunity.visualstudio.com/t/error-c2760-in-combaseapih-with-windows-sdk-81-and/185399
+            tools.replace_in_file(
+                os.path.join(self._source_subfolder, "meson.build"),
+                "cpp_std=c++", "cpp_std=vc++")
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def build(self):
+        self._patch_sources()
+        with tools.environment_append(tools.RunEnvironment(self).vars):
+            meson = self._configure_meson()
+            meson.build()
+
+    def _configure_meson(self):
+        meson = Meson(self)
+        defs = {
+            "build-documentation": "false",
+            "msvc14x-parallel-installable": "false",
+            "default_library": "shared" if self.options.shared else "static",
+        }
+
+        meson.configure(
+            defs=defs,
+            build_folder=self._build_subfolder,
+            source_folder=self._source_subfolder,
+            pkg_config_paths=[self.install_folder],
+        )
+
+        return meson
+
+    def package(self):
+        self.copy("COPYING", dst="licenses", src=self._source_subfolder)
+        meson = self._configure_meson()
+        meson.install()
+
+        shutil.move(
+            os.path.join(self.package_folder, "lib",
+                         f"pangomm-{self._api_version}", "include",
+                         "pangommconfig.h"),
+            os.path.join(self.package_folder, "include",
+                         f"pangomm-{self._api_version}", "pangommconfig.h"))
+
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(
+            os.path.join(self.package_folder, "lib",
+                         "pangomm-{self._api_version}", "include"))
+
+        if is_msvc(self):
+            tools.remove_files_by_mask(
+                os.path.join(self.package_folder, "bin"), "*.pdb")
+            if not self.options.shared:
+                rename(
+                    self,
+                    os.path.join(self.package_folder, "lib",
+                                 f"libpangomm-{self._api_version}.a"),
+                    os.path.join(self.package_folder, "lib",
+                                 f"pangomm-{self._api_version}.lib"),
+                )
+
+
+    def package_info(self):
```

Comment:
If the only difference between those versions are the version numbers, then make the version numbers variables and simplify this

Reply:
The version of glibmm is different as well.

I thought this is clearer in the expense of a little redundancy